tests/x11/toolkits/fltk.pm

```diff
@@ -21,8 +41,8 @@ sub run {
     script_run './fltk', 0;
```

Comment:
I would prefer to fix the root cause, because it looks that the test is trying to workaround a problem with the code in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/toolkits/fltk.cc

Reply:
Yes, this is just workaround to make sure the label can show correctly. The fltk.cc file uses very simple example to test fltk.  Just opened one window and set one box with 'Hello Word: FLTK' label. It may related with virtual machine graphics driver. Sometimes the label cannot show up.

Reply:
I tried to use Fl_Button to test fltk, But it also needs to click button several times then the 'label' can show up.